### ISSUE TITLE
Learn.md 

### ISSUE THREAD

**Issue by Varda003** [TusharAMD/SuperSpeechSaga#4]
As mentioned in GSSOC'24  learn.md file necessary to add. I can create it

**Comment by Varda003** [comment]
@TusharAMD please label the issue and also add GSSOC'24 otherwise Pr wont be count please.

**Comment by akshatsharma2407** [comment]
Hello, I am fully capable of updating your LEARN section as a GSSoC contributor. Kindly assign me this issue as I possess intermediate knowledge of the technology utilized in your project. I am confident that I am the best fit for this task.

**Comment by Varda003** [comment]
HI @TusharAMD  can you please add label of GSSOC'24 and level as well cause as mentioned by the sanjay-kv my  contribution won't be counted.

**Comment by Varda003** [comment]
hi @TusharAMD I already done with creating Learn.md file please adds labels so I can create a pull request
![Screenshot 2024-05-11 114452](https://github.com/TusharAMD/SuperSpeechSaga/assets/136989588/e5dcd4b2-6b96-4d60-81f7-d36cc955072e)

**Comment by Varda003** [comment]
all the things above are mentioned briefly giving complete info

**Comment by TusharAMD** [comment]
@Varda003 Done. Will be adding levels labels soon

**Comment by Varda003** [comment]
Sir @TusharAMD Its completely upto you what about labeling this issue as 1 , 2 , 3 but please it will be great if you could consider it level 2 issue as I had create whole Learn.md file own my own and it took great efforts to go all through your project so that I don't miss out anything. Please consider it.

**Comment by TusharAMD** [comment]
@Varda003 Hi, is it done? Can you please make PR if done?

**Comment by Varda003** [comment]
Yes I already made the Pr @TusharAMD . Please assign the level 2 label as well. It took me lot of time create this file.
